### ISSUE TITLE
Unable to install FoE Helper extension on Chrome

### ISSUE THREAD

**Issue by Coumbayah** [mainIine/foe-helfer-extension#2850]
Error received using the Chrome Store to attempt to install FoE Helper v3.6.5

"Download error:  Could not move extension into directory profile"

Anyone can help or direct me towards instructions on how fix the issue?
Thank you!

**Comment by outoftheline** [comment]
when googling the error, it looks like a chrome issue. 
please look for solutions on the internet.

**Comment by Coumbayah** [comment]
Thank you... I googled and found this fix that worked for me.  Fix #6
https://windowsreport.com/chrome-could-not-move-extension-directory-into-profile/#:~:text=An%20error%20has%20occurred%20could,or%20rename%20its%20Default%20directory.

--> inside %localpath% > Google > Chrome > rename folder "Default" to "Default-backup" and cut-paste it somewhere else.  Once reopening the browser, I re-signed on my Google account to sync my bookmarks, preferences, settings, etc.

I went to the Chrome Store and was able to install FoE Helper without any issues.  Cheers!
